merlin-rxjava/src/test/java/com.novoda.merlin.rxjava/MerlinObservableShould.java

```diff
@@ -0,0 +1,103 @@
+package com.novoda.merlin.rxjava;
+
+import android.content.Context;
+
+import com.novoda.merlin.Merlin;
+import com.novoda.merlin.NetworkStatus;
+import com.novoda.merlin.registerable.bind.Bindable;
+import com.novoda.merlin.registerable.connection.Connectable;
+import com.novoda.merlin.registerable.disconnection.Disconnectable;
+
+import java.util.List;
+
+import org.junit.Test;
+
+import org.mockito.ArgumentCaptor;
+
+import rx.Subscription;
+import rx.observers.TestSubscriber;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class MerlinObservableShould {
+
+    @Test
+    public void unbindWhenUnsubscribed() {
+        Merlin merlin = mock(Merlin.class);
+        Subscription subscription = MerlinObservable.from(merlin)
+                                                    .subscribe();
+        subscription.unsubscribe();
+
+        verify(merlin, times(1)).unbind();
+    }
+
+    @Test
+    public void receiveOneAvailableNetworkStatusStateOnConnect() {
+        Merlin merlin = mock(Merlin.class);
+        ArgumentCaptor<Connectable> argumentCaptor = ArgumentCaptor.forClass(Connectable.class);
+        TestSubscriber<NetworkStatus.State> testSubscriber = new TestSubscriber<>();
+        MerlinObservable.from(merlin)
```

Comment:
I think `MerlinObservable.from(merlin).test()` has a nicer api. You can consider using it.